ner_v2/tests/numeral/number/en/test_number_detection.py

```diff
@@ -148,12 +148,12 @@ def test_en_number_detection_for_integer_number(self):
         self.assertEqual(len(zipped), 1)
         self.assertIn(({'value': '100', 'unit': None}, u'100'), zipped)
 
```

Comment:
Missing tests for`unit_type=None` or a different unit is given

For e.g.

`my monthly salary is 10.12k rupees`, `unit_type=None` must return [`10120`]
`my monthly salary is 10.12k rupees`, `unit_type='weight'` must return []